io.github.jazztickets.irrlamb.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.jazztickets.irrlamb
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: irrlamb
+rename-desktop-file: irrlamb.desktop
+rename-icon: irrlamb
+
+modules:
+  - name: irrlamb
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/jazztickets/irrlamb/releases/download/v1.0.2/irrlamb-1.0.2r648-src.tar.gz
```

Comment:
it seems that by default this tries to build with `-mfpmath=sse`... which of course doesn't work on non x86_64.

Reply:
SSE is for floating point determinism in the physics and replay validation system. The game doesn't need to run on ARM.

Reply:
I know what SSE is. And `-mfpmath=sse` is default on x86_64 anyway.